resources/function_help/json/if

```diff
@@ -4,10 +4,11 @@
   "description": "Tests a condition and returns a different result depending on the conditional check.",
   "arguments": [
 	{"arg":"condition","description":"the condition which should be checked"},
-	{"arg":"result_when_true","description":"the result which will be returned when the condition is True"},
-	{"arg":"result_when_false","description":"the result which will be returned when the condition is False"}
+	{"arg":"result_when_true","description":"the result which will be returned when the condition returns True or not null value"},
```

Comment:
> About the not null value, it wasn't just about null but also about zero (0) value (which is a valeur nulle in french).

In English they have very strict meanings which cannot be mixed and the two things need to be treated differently in the docs. I hope the French translation of the docs do also ;)

I'd start with some comparison

```
if (1+1=2, 'Yes', 'No') --> 'Yes'
if (1+1=3, 'Yes', 'No') --> 'No'
if (5 > 3, 'Yes', 'No') --> 'Yes'
```
and then continue with all the others.